### PR TITLE
Create 2024-05-22

### PR DESCRIPTION
**Pull request by mashaeva-va**
Updating IDs for Health APIs

### CODE REVIEW

File: content/health/clinical-fhir-api/release-notes/2024-05-22 [hunk 1]
```diff
@@ -0,0 +1 @@
+FHIR logical ID values have been updated. No impact to users. Previous FHIR logical IDs remain valid.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-05-22:1]:
```suggestion
We've updated the FHIR logical ID values. This has no impact to users. Previous FHIR logical IDs remain valid.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Some small changes.

File: content/health/clinical-fhir-api/release-notes/2024-05-22 (hunk 1 above)
Comment (mashaeva-va) [reply content/health/clinical-fhir-api/release-notes/2024-05-22:1]: Agreed!

File: content/health/clinical-fhir-api/release-notes/2024-05-22 [hunk 2]
```diff
@@ -0,0 +1 @@
+We've updated the FHIR logical ID values. This has no impact to users. No impact to users. Previous FHIR logical IDs remain valid.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-05-22:1]:
```suggestion
We've updated the FHIR logical ID values. This has no impact to users. Previous FHIR logical IDs remain valid.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
Getting rid of a sentence

File: content/health/clinical-fhir-api/release-notes/2024-05-22 [hunk 3]
```diff
@@ -0,0 +1 @@
+We've updated the FHIR logical ID values. This has no impact to users. Previous FHIR logical IDs remain valid.
```
Comment (joeabraun) [inline content/health/clinical-fhir-api/release-notes/2024-05-22:1]:
```suggestion
We've updated the FHIR logical ID values. This has no impact on users. Previous FHIR logical IDs remain valid.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
One last quick change

**Review by joeabraun — APPROVED** [review]
LGTM!

**Pull request merged**